src/store/audioContinuousPlayer.ts

```diff
@@ -111,19 +111,25 @@ export class ContinuousPlayer extends EventTarget {
 
     await this.promise;
   }
-}
 
-export interface ContinuousPlayer extends EventTarget {
   addEventListener<K extends keyof ContinuousPlayerEvents>(
     type: K,
     listener: (this: ContinuousPlayer, ev: ContinuousPlayerEvents[K]) => void,
-    options?: boolean | AddEventListenerOptions
+    options?: boolean | AddEventListenerOptions,
   ): void;
   addEventListener(
     type: string,
     listener: EventListenerOrEventListenerObject,
-    options?: boolean | AddEventListenerOptions
+    options?: boolean | AddEventListenerOptions,
   ): void;
+
+  addEventListener(
+    type: string,
+    listener: EventListenerOrEventListenerObject,
+    options?: boolean | AddEventListenerOptions,
+  ) {
+    super.addEventListener(type, listener, options);
+  }
 }
```

Comment:
この辺り冗長な気がしますね。
ちょっとコメント書いとくのどうでしょう。

```suggestion
  // FIXME: 冗長かも？
  addEventListener(
    type: string,
    listener: EventListenerOrEventListenerObject,
    options?: boolean | AddEventListenerOptions,
  ) {
    super.addEventListener(type, listener, options);
  }
}
```

@cm-ayf 
ちょっとTypeScriptに詳しくなく、もしご存知でしたらお聞きしたいことが 🙇 
`addEventListener`のtypeに型の制約を加える処理だと思うのですが、これってシグネチャの定義だけで行けたりするんでしょうか？
プルリクエストの実装の場合superを呼び出しているからなんか冗長かもと思いつつ、かと言って良いやり方もわからないなぁと･･･。